Fixed orthography in ReadMe file (#693)
savushkin-r-d/EasyEPLANner: ReadMe.md

# EasyEPLANner - Open Source

### Repository

This repository is an open source project - EasyEPLANner.
We are working on the project and solving various issues related to the development and project life.

### EasyEPLANner

> <img src="docs/user_manual/images/EasyEPLANnerPreview.png">

Reviewer: ```suggestion
<img src="docs/user_manual/images/EasyEplannerPreview.png">
```

Сломали картинку.

Reviewer 2: Можно тогда и картинку переименовать, чтобы ссылка корректная была.

Reviewer: ОК. Переименовать картинку по новому регистру и не принимать регистр, который предложил я. Тогда подойдут правки.

> The EasyEPLANner is developing as an Add-In to EPLAN 2.9 version. The add-in is using for EPLAN projects and helps you to reduce the work of an Automation engineer and a Software engineer as well. 

Reviewer 2: ```suggestion
The EasyEPLANner is being developed as an Add-In to EPLAN 2.9 version. The add-in is using for EPLAN projects and helps you to reduce the work of an Automation engineer and a Software engineer as well. 
```

### Contribution
If you want to contribute to the development of our project then check out [how to do it better](docs/contributing.md) before you start.

### Feedback

Feel free to contact us via Slack or Google group.

> * [Google group](https://groups.google.com/forum/#!forum/easyEPLANner);

Reviewer: ```suggestion
* [Google group](https://groups.google.com/forum/#!forum/easyeplanner);
```
Сломана ссылка была

Reviewer: В принципе, для веба не важно - можно так оставить.

> * Channel in [Slack](https://slack.com) - easyEPLANner.slack.com.

Reviewer: ```suggestion
* Channel in [Slack](https://slack.com) - easyeplanner.slack.com.
```
Некорректный адрес задан
